cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -113,8 +114,25 @@ public void create(HttpRequest request, HttpResponder responder, @PathParam("nam
       }
 
       NamespaceConfig config = metadata.getConfig();
-      if (config != null && !Strings.isNullOrEmpty(config.getSchedulerQueueName())) {
-        builder.setSchedulerQueueName(config.getSchedulerQueueName());
+      if (config != null) {
```

Comment:
The  logic of handling the config is in Admin. Here we are just creating the NamespaceMeta from what was  provided. Since the Handler takes the  id of namespace and metadata as body and recreates the meta with the  namespace id coming from the path param we are just doing that here. Does it looks wrong ?